custom_components/wyzeapi/light.py

```diff
@@ -94,12 +109,12 @@ def brightness(self):
         This method is optional. Removing it indicates to Home Assistant
         that brightness is not supported for this light.
         """
-        return self.__light.brightness
+        return self.translate(self.__light.brightness, 1, 100, 1, 255)
 
     @property
     def color_temp(self):
         """Return the CT color value in mired."""
-        return self.__light.color_temp
+        return self.translate(self.__light.color_temp, 2700, 6500, 500, 140)
```

Comment:
There's a lot of repeating integers hanging around in this file. Maybe break it out into functions `translate_brightness` and `translate_color_temp`? However, it appears that this color temp translation params are different from the ones below on line 135... is this intentional?

Reply:
Yeah it is. It makes sure the changes go both ways.